Logger/FullName.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Logger;
+
+public record FullName
+{
+
+    // We decided to handle the full name storage this way to handle middle names being in the name or not
+    // We are allowing the name to be mutable because names can be changed
+    private string fullName;
+
+    public FullName(string fName, string mName, string lName)
```

Comment:
we don't have to worry about longer names having some performance boost or something. full names are more clear.\
```suggestion
    public FullName(string firstName, string middleName, string lastName)
```